spacewalk/setup/lib/Spacewalk/Setup.pm

```diff
@@ -921,9 +921,6 @@ sub postgresql_reportdb_setup {
     if (-e Spacewalk::Setup::DEFAULT_RHN_CONF_LOCATION) {
         my %dbOptions = ();
         read_config(Spacewalk::Setup::DEFAULT_RHN_CONF_LOCATION, \%dbOptions);
-        foreach my $key (keys %dbOptions) {
-            delete $dbOptions{$key} if ($key !~ /^report_/);
-        }
```

Comment:
Have you tried this? how will the template look like with this change. Please write a bit more info in the issue to explain the consequences of the change

Reply:
Thanks Michael, previous changes were wrong. We need to delete all the keys, in this way `/var/lib/rhn/rhn-satellite-prep/etc/rhn/rhn.conf` contains just variable name and not values. After running this procedure https://www.uyuni-project.org/uyuni-docs/en/uyuni/administration/troubleshooting/tshoot-retrying-setup-target-system.html the file doesn't change.

Without this changes `/var/lib/rhn/rhn-satellite-prep/etc/rhn/rhn.conf` contains value instead of variable name for `report_db` variable. Moreover, each new setup creates a new file with new report db password (the previous file is moved to `rhn.conf.orig` )